src/test/java/edu/ksu/canvas/attendance/repository/AttendanceRepositoryImplITest.java

```diff
@@ -31,32 +32,50 @@
 @RunWith(SpringJUnit4ClassRunner.class)
 public class AttendanceRepositoryImplITest {
 
+    private static final long CANVAS_SECTION_ID = 500L;
+
     @Autowired
     private AttendanceRepository attendanceRepository;
     
     @Autowired
     private AttendanceStudentRepository studentRepository;
     
-    private Attendance existingAttendance; 
+    private Attendance existingAttendance;
+    private Attendance attendance1;
+    private Attendance attendance2;
     
     
     @Before
     public void setup() throws ParseException {
         SimpleDateFormat sdf = new SimpleDateFormat("MM/dd/yyyy");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "MM/dd/yyyy" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)